packages/superset-ui-plugin-chart-table/src/Table.tsx

```diff
@@ -0,0 +1,276 @@
+        maxes[metrics[i]] = Math.max(...dataArray[metrics[i]].map(Math.abs));
+      } else {
+        maxes[metrics[i]] = Math.max(...dataArray[metrics[i]]);
+        mins[metrics[i]] = Math.min(...dataArray[metrics[i]]);
+      }
+    }
+
+    // const tsFormatter = getTimeFormatter(tableTimestampFormat);
+    let formattedData = data.map(row => ({
+      data: row,
+    }));
+
+    if (sortByKey) {
+      formattedData = formattedData.sort((a, b) => {
+        const delta = a.data[sortByKey] - b.data[sortByKey];
+        if (orderDesc) {
+          return -delta;
+        }
+        return delta;
+      });
+      if (metrics.indexOf(sortByKey) < 0) {
+        formattedData = formattedData.map(row => {
+          const data = { ...row.data };
+          delete data[sortByKey];
+          return {
+            data,
+          };
+        });
+      }
+    }
+
+    const tsFormatter = getTimeFormatter(tableTimestampFormat);
+
+    const heightType = 'small';
+    const getRenderer = (column: ColumnType) => (props: RendererProps) => {
+      const { key } = props;
+      let value = props.row.rowData.data[key];
+      const cellHash = `${key}#${value}`;
+      const isMetric = metrics.indexOf(key) >= 0;
+      let Parent;
+
+      if (isMetric) {
+        let left = 0;
+        let width = 0;
+        if (alignPositiveNegative) {
+          width = Math.abs(Math.round((value / maxes[key]) * 100));
+        } else {
+          const posExtent = Math.abs(Math.max(maxes[key], 0));
+          const negExtent = Math.abs(Math.min(mins[key], 0));
+          const tot = posExtent + negExtent;
+          left = Math.round((Math.min(negExtent + value, negExtent) / tot) * 100);
+          width = Math.round((Math.abs(value) / tot) * 100);
+        }
+        const color = colorPositiveNegative && value < 0 ? NEGATIVE_COLOR : POSITIVE_COLOR;
+        Parent = ({ children }: { children: React.ReactNode }) => {
+          const boxStyle: CSSProperties = {
+            margin: '0px -16px',
+            backgroundColor: tableFilter && selectedCells.has(cellHash) ? '#ffec99' : undefined,
+          };
+          const boxContainerStyle: CSSProperties = {
+            position: 'relative',
+            height: HEIGHT_TO_PX[heightType],
+            textAlign: isMetric ? 'right' : 'left',
+            display: 'flex',
+            alignItems: 'center',
+            margin: '0px 16px',
+          };
+          const barStyle: CSSProperties = {
+            background: color,
+            width: `${width}%`,
+            left: `${left}%`,
+            position: 'absolute',
+            height: 24,
+          };
+
+          return (
+            <div style={boxStyle}>
+              <div style={boxContainerStyle}>
+                <div style={barStyle} />
+                <div style={{ zIndex: 10, marginLeft: 'auto' }}>{children}</div>
+              </div>
+            </div>
+          );
+        };
+
+        if (key[0] === '%') {
```

Comment:
I also wonder if we can find better way to configure this that this hard coded rule